deprecated/azure/misc/azure_ha_test.py

```diff
@@ -0,0 +1,414 @@
+#!/usr/bin/env python
```

Comment:
Please rename the file to azure_ha_test_python2.py
Since in the future there will be also deprecated python 3 ha_test and we need to differentiate